Doc/faq/library.rst

```diff
@@ -181,7 +181,7 @@ How do I create documentation from doc strings?
 The :mod:`pydoc` module can create HTML from the doc strings in your Python
 source code.  An alternative for creating API documentation purely from
 docstrings is `epydoc <http://epydoc.sourceforge.net/>`_.  `Sphinx
```

Comment:
This only has `http` -- https://epydoc.sourceforge.net/ fails.

Reply:
Looks like the `https` link works now

Reply:
Confirm. Do you mind to create a pull request?

Reply:
Looks like it has already been fixed in a different PR https://github.com/python/cpython/commit/dd53b79de0ea98af6a11481217a961daef4e9774

My mistake!